app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -22,8 +23,18 @@ angular.module("openshiftConsole")
         }, {
           value: "Pi",
           label: "PiB"
-        }];
+          }];
         scope.claim.selectedLabels = [];
+        DataService.get({
+          group: 'storage.k8s.io',
+          resource: 'storageclasses'
+        }, null, {}).then(
+          function(storageClasses) { // success
+            _.each(storageClasses.items, function(item) {
```

Comment:
switching to use list will change the data you get back, this will need to change to storageClasses.by('metadata.name)

also instead of calling this item, call it storageClass